### ISSUE TITLE
Только русские ip-адресса через vpn

### ISSUE THREAD

**Issue by xScr3amox** [zhovner/zaborona_help#208]
Здравствуйте, есть возможность сделать так что бы только подключение к всем русским сайтам проходило через впн, а остальной мир без впн?

**Comment by toxi22** [comment]
Здравствуйте.
Подумаем, как лучше организовать доступ. На данный момент, пока не планировалось весь РФ сегмент пускать через впн.

**Comment by xScr3amox** [comment]
Добавьте хотя бы онлайн кинотеатры типа rezka.ag, и читалки типа MyBook. С начала войны начались проблемы с функционалом этих приложений, без впн ни там ни там ничего не грузится, а для кого-то эти сайты не плохой способ не сойти с ума во время войны, хотя бы как то отвлечься.

**Comment by xScr3amox** [comment]
Сами сайты вроде как грузятся, но контент на них - нет.

**Comment by toxi22** [comment]
> Добавьте хотя бы онлайн кинотеатры типа rezka.ag, и читалки типа MyBook. С начала войны начались проблемы с функционалом этих приложений, без впн ни там ни там ничего не грузится, а для кого-то эти сайты не плохой способ не сойти с ума во время войны, хотя бы как то отвлечься.

Добавили некоторые маршруты на все сервера

**Comment by xScr3amox** [comment]
> > Сами сайты вроде как грузятся, но контент на них - нет.
> > роуты rezka.ag: 176.58.48.0/23 176.58.48.0-176.58.49.255
> 
> Добрый день. Добавили. Переподключитесь и проверьте работу.

К сожалению не работает, все равно(

**Comment by xScr3amox** [comment]
Сайт rezka.ag официально прокомментировал, что для возможности просмотра их контента нужно разблокировать следующие роуты:

176.58.38.0/24
176.58.39.0/24
176.58.40.0/24
176.58.41.0/24
176.58.42.0/24
176.58.43.0/24
176.58.44.0/24
176.58.48.0/24
176.58.45.0/24
176.58.46.0/24
176.58.47.0/24
176.58.48.0/24
176.58.49.0/24
176.58.50.0/24
176.58.51.0/24
176.58.52.0/24
176.58.53.0/24
176.58.54.0/24
176.58.57.0/24
185.190.188.0/24
185.190.189.0/24
185.190.190.0/24
185.190.191.0/24

Список доменов, которые используются для трансляции видео:
stream.voidboost.cc и его поддомены *.stream.voidboost.cc
stream.voidboost.in и его поддомены *.stream.voidboost.in

Писал своему провайдеру, те ничего сделать не могут, так что осталась надежда на вас.

**Comment by toxi22** [comment]
> Сайт rezka.ag официально прокомментировал, что для возможности просмотра их контента нужно разблокировать следующие роуты:

Добавили. Просьба проверить, есть ли изменения либо нет, в работе сайта

**Comment by xScr3amox** [comment]
Заработало! Большущее вам спасибо!)

**Comment by b0rman** [comment]
Здравствуйте.
Использую Rezka.ag для просмотра фильмов через HDVideoBox - к сожалению, видео не проигрывается, хотя постеры и описание загружаются корректно. Возможно, поменялись IP?

> Список доменов, которые используются для трансляции видео:
> stream.voidboost.cc и его поддомены *.stream.voidboost.cc
> stream.voidboost.in и его поддомены *.stream.voidboost.in

**Comment by toxi22** [comment]
> Здравствуйте. Использую Rezka.ag для просмотра фильмов через HDVideoBox - к сожалению, видео не проигрывается, хотя постеры и описание загружаются корректно. Возможно, поменялись IP?
> 
> > Список доменов, которые используются для трансляции видео:
> > stream.voidboost.cc и его поддомены *.stream.voidboost.cc
> > stream.voidboost.in и его поддомены *.stream.voidboost.in

Добрый день.
Проверим

**Comment by toxi22** [comment]
Здравствуйте.
Проблема все еще актуальна?

**Comment by b0rman** [comment]
Да, к сожалению, у меня вообще перестала показывать HDVideoBox с Rezka. В то время, как на другом провайдере, которые не блочит IP показывает отлично.
